packages/react/src/views/Message/MessageToolbox.js

```diff
@@ -130,7 +130,10 @@ export const MessageToolbox = ({
         id: 'edit',
         onClick: () => handleEditMessage(message),
         iconName: 'edit',
-        visible: isAllowedToEditMessage,
+        visible:
+          isAllowedToEditMessage &&
+          message.files?.[0].type !== 'audio/mpeg' &&
```

Comment:
Instead of checking here directly, make a variable for this check and use it here.

Reply:
Done. Thank you, Zishan.